collections/Ashmole/MS_Ashmole_1517.xml

```diff
@@ -38,37 +41,90 @@
-                     <origDate calendar="Gregorian" when="1400">c. 1400</origDate>
                      <origPlace>
-                        <country key="place_7002445">English</country>
+                        <country key="place_7002445">England</country>
                      </origPlace>
+                     <origDate calendar="Gregorian" notAfter="1425" notBefore="1400">15th century, first quarter</origDate>
+                     <p>Originally the manuscript may have contained a calendar-lectionary at the beginning and a complete New Testament. The Old Testament lectionary is similar to that in MS. Selden Supra 51.</p>
                   </origin>
+                  <provenance><q>john m(?)[…]th</q>
+                     <locus>(fol. 64r)</locus>, possibly a reference to a missing beginning of St John’s gospel, 16th century (?).</provenance>
+                  <provenance><q>Edmu[n]d peyson</q>, 16th century <locus>(fol. 143r)</locus>.</provenance>
+                  <provenance>Flourished ‘I’ with the letters T, H, R, A, V written inside it, 16th century <locus>(fol. 192r)</locus>.</provenance>
+                  <provenance>Pen trials and names in several 16th-century hands, including: <q>Hary Danyell of sancte martyns farmar vnto John Gylbard of London</q>; <q>wyllym storky, dunston chaypman</q>, <q>william breges</q>
+                     <locus>(fol. 192v)</locus>.</provenance>
+                  <provenance>Elias Ashmole (1617–1692), see <title>Oxford Dictionary of National Biography</title>: transcript of missing passages (see Text); autograph on fol. 1r.</provenance>
+                  <provenance>Ashmolean Museum, Oxford: donated by Ashmole to Oxford University in 1677.</provenance>
+                  <acquisition>Bodleian Library: transferred from the Ashmolean Museum in 1860.</acquisition>
                </history>
                <additional>
                   <adminInfo>
                      <recordHist>
-                        <source><ref target="https://medieval.bodleian.ox.ac.uk/about">Summary description</ref> abbreviated from the Quarto Catalogue (W. H. Black, <title>A descriptive, analytical, and critical catalogue of the manuscripts bequeathed unto the University of Oxford by Elias Ashmole Esq....</title>, Quarto Catalogues X, 1845). <listBibl>
+                        <source>Elizabeth Solopova, <title><ref target="https://jstor.org/stable/j.ctt1gn6cmr">Manuscripts of the Wycliffite Bible in the Bodleian and Oxford College Libraries</ref></title>, Exeter Medieval Texts and Studies (Liverpool: Liverpool University Press, 2016), no. 1. Previously described in the Quarto Catalogue (W. H. Black, <title>A descriptive, analytical, and critical catalogue of the manuscripts bequeathed unto the University of Oxford by Elias Ashmole Esq....</title>, Quarto Catalogues X, 1845). <listBibl>
                               <bibl facs="aeh0714.gif" type="QUARTO">Quarto Catalogue, cols. 1417–18</bibl>
```

Comment:
perhaps worth clarifying what has / hasn't been included (everything except her discussion of dialect?). Perhaps worth adding note in textLang that she provides more detail re dialect?

Reply:
I suppose we might as well include it: I've tried shoehorning dialect in under `<origin>`, unless you can think of something better. As it is technical I think it would put the reader off to have it right at the top in `<textLang>`.